pkgs/by-name/cm/cmake/package.nix

```diff
@@ -178,10 +180,16 @@ stdenv.mkDerivation (finalAttrs: {
 
   doCheck = false; # fails
 
-  passthru.updateScript = gitUpdater {
-    url = "https://gitlab.kitware.com/cmake/cmake.git";
-    rev-prefix = "v";
-    ignoredVersions = "-"; # -rc1 and friends
+  passthru = {
+    tests = import ./test {
```

Comment:
List the tests explicitly instead of creating a wall of code.

Reply:
Addressed.